ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -140,7 +149,12 @@ extension RACCommand {
 			// FIXME: Workaround for <~ being disabled on SignalProducers.
 			|> startWithSignal { signal, disposable in
 				let bindDisposable = enabledProperty <~ signal
-				disposable.addDisposable(bindDisposable)
+				
```

Comment:
This is repeated in a few places. How do you feel about something like `bindDisposable.disposeAfterSignalTermination(signal)`?